mautic/mautic
# Prevent run report sheduler If already run

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | yes
| New feature/enhancement? (use the a.x branch)      | no
| Deprecations?                          | no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              |  no 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


If you use send reports to email, and your report processing take more time like your cron timeout mautic:report:scheduler, command start process same report again. This make issue on big reports
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create report with large data
3. Use export & send
4. Try running the example-command, and a few seconds later, run the same example-command again in parallel using separate terminal windows or background processes
5. After this PR you should see Script in progress. Can force execution by using --force.
